include/networkit/io/DibapGraphReader.hpp

```diff
@@ -8,26 +8,13 @@
 #ifndef DIBAPGRAPHREADER_H_
 #define DIBAPGRAPHREADER_H_
 
-#if !defined _WIN32 && !defined _WIN64 && !defined WIN32 && !defined WIN64
+#ifndef DNETWORKIT_WINDOWS
```

Comment:
This macro is called `NETWORKIT_WINDOWS`, isn't it?

Reply:
Yes!